lib/ramble/docs/configuration_files.rst

```diff
@@ -624,4 +626,27 @@ format is as follows:
       cross_reference_var: 'var in <app>.<workload>.<exp>'
 
 Variables can be defined as lists, scalars, or can refer to a variable defined in
-another fully qualified experiment (through the ``cross_ref_var`` syntax).
+another fully qualified experiment (through the syntax shown in ``cross_reference_var``).
+
+For more information on variable expansion rules, see:
+:ref:`workspace variable dictionary definitions<variable-dictionaries>`.
+
+.. _variants-config:
+
+----------------
+Variants Section
+----------------
+
+The variants config section is used to customize variants to the experiment creation.
+These can include application defined variants, or higher level Ramble provided
+variants. Currently, the only supported variants is ``package_manager`` which
```

Comment:
What's the default behavior currently? Does it default to use spack if no variant config is set?

Reply:
Ah, good point. Let me add that into the documentation.